src/app/tests/suites/certification/Test_TC_CDOCONC_1_1.yaml

```diff
@@ -42,15 +42,12 @@ tests:
     - label:
           "Step 3a: Read the global attribute: FeatureMap and check for either
           bit 0 or 1 set"
-      PICS:
-          " !CDOCONC.S.F00 && !CDOCONC.S.F01 && !CDOCONC.S.F02 && !CDOCONC.S.F03
-          && !CDOCONC.S.F04 && !CDOCONC.S.F05"
       command: "readAttribute"
       attribute: "FeatureMap"
       response:
-          value: 0
           constraints:
               type: bitmap32
+              hasMasksSet: [0x03]
```

Comment:
Doesn't this check for both 0 and 1 set?

Reply:
It used to. Boris pointed out that the C++ version returns true for any of the bits set and we should align for Python (see #27248). 

I tested it with bit 0 set and with bit 1 set and it passes.

Reply:
FWIW, Thomas has raised an issue to make the naming better for this: https://github.com/project-chip/connectedhomeip/issues/28837